docs/router-tutorial/react-navigation-stack/react-navigation-stack.md

```diff
@@ -0,0 +1,338 @@
+---
+title: react-navigation-stack
+parent: React Navigation
+nav_order: 1
+has_children: false
+---
+
+# **React Navigation**
+
+React Navigation (RN) - related libraries that help you move between screens.
```

Comment:
Actually, the use of RN can be confusing between React Native or React Navigation. 
Generally, RN is more suitable for React Native.

Reply:
Delete (RN)

Reply:
Oh this is already removed & outdated.